lib/Storage/MongoDbTimeSeriesStore.php

```diff
@@ -120,137 +127,231 @@ public function setDependencies($log, $date, $layoutFactory = null, $campaignFac
     /** @inheritdoc */
     public function addStat($statData)
     {
-        foreach ($statData as $k => $stat) {
 
-            $statData[$k]['statDate'] = new UTCDateTime($statData[$k]['statDate']->format('U') * 1000);
-            $statData[$k]['start'] = new UTCDateTime($statData[$k]['fromDt']->format('U') * 1000);
-            unset($statData[$k]['fromDt']);
+        $statData['statDate'] = new UTCDateTime($statData['statDate']->format('U') * 1000);
```

Comment:
we could take this opportunity to comment why we're doing this, and why we're doing the unset